packages/genesys-spark-components/README.md

```diff
@@ -15,7 +15,7 @@ At any given time there are three types of components present in the library:
 - **beta**: New components where the API design is still being explored. Breaking changes _may_ happen without a major version change.
 - **legacy**: Old components that have been replaced by a new component, or a newer version of the component with an incompatible API. These will be removed in the next major release.
 
-For more details on the component evolution process see the full [documentation on the topic](./packages/genesys-spark-components/documentation/COMPONENT_EVOLUTION.md)
+For more details on the component evolution process see the full [documentation on the topic](../../packages/genesys-spark-components/documentation/COMPONENT_EVOLUTION.md)
```

Comment:
Something I noticed within the component evolution doc is that the cycle is noted as `Beta` , `Supported` , `Legacy` . I am just wondering should this be `Beta`, `Stable` , `Legacy`